css/style.css

```diff
@@ -0,0 +1,17 @@
+.container{
+    width:500px;
+    margin:100px auto;
+}
+
+*{
+    font-size:20px;
+}
+ul{
+    list-style:none;
+    text-align:right;
+    width:200px;
+    margin:10px; 
+}
+li.checked{
+    text-decoration:line-through;
+}
```

Comment:
Newline at the end is not really important, but slightly preferred. Also I don't often see no space between the : and the value in CSS, but again, not really important. May improve readability though.